packages/tomls/src/omnibus/reya_cronos.toml

```diff
@@ -89,6 +94,13 @@ redstonePriceFeedETH = "0x187078471869387119FB88E8F033412A0F0fdfcd"
 redstonePriceFeedBTC = "0x22Ef8eb05189cF39a347F93a33508D0ce9e37830"
 redstonePriceFeedSOL = "0x3e3762c10F018fE02701be651b5c0937C3a7d7e1"
 
+[var.stork_asset_pair_ids]
+storkAssetPairIdSOL = "SOLUSD"
```

Comment:
are these defined by stork? if not shall we add a dash between the 2 tokens?

Reply:
can you ask ab ?

Reply:
they're defined by stork